### ISSUE TITLE
Can u support VBoxLayout for ButtonWithIcon

### ISSUE THREAD

**Issue by owner888** [fyne-io/fyne#2438]
I would like to dynamically set VBoxLayout for ButtonWithIcon like the down picture，but the button.layout is private，when I change source HBoxLayout to VBoxLayout，the button theme is Not good looking。
### Describe the solution you'd like to see:

![111111](https://user-images.githubusercontent.com/4317332/132338103-5b01feab-d58c-48f8-9437-2881bbea508f.png)

**Comment by andydotxyz** [comment]
That looks like the `AppTabs` container, as it controls items in the area below the buttons

**Comment by andydotxyz** [comment]
The butttons and tab widgets have distinct styles so that users know what will likely happen when interacted.

**Comment by owner888** [comment]
> That looks like the `AppTabs` container, as it controls items in the area below the buttons

Yes I think so, but NewTabItemWithIcon does not support VBoxLayout like button, So I wonder if it’s easier to change the button

**Comment by andydotxyz** [comment]
It depends on the orientation and the output device (for example the vertical layout is standard when AppTabs renders on mobile).

**Comment by owner888** [comment]
> It depends on the orientation and the output device (for example the vertical layout is standard when AppTabs renders on mobile).

oh thank u, but can u support setting for user to choice ^_^

**Comment by owner888** [comment]
> It depends on the orientation and the output device (for example the vertical layout is standard when AppTabs renders on mobile).
Unable to pass layout parameters when I use NewTabItemWithIcon function
func NewTabItemWithIcon(text string, icon fyne.Resource, content fyne.CanvasObject)

**Comment by owner888** [comment]
> To summarise the initial request adding vertical to standard button would break alignment in standard contexts.
> However perhaps this is a feature request for text under toolbar buttons perhaps? The toolbar can have custom content in it too, so this would satisfy the requested features in the screenshot I think?

ok, i got it, thanks bro
